packages/clay-tooltip/src/ClayTooltip.js

```diff
@@ -187,7 +188,48 @@ class ClayTooltip extends ClayComponent {
 	}
 
 	/**
-	 * Handles transionend event.
+	 * Handles touchstart events.
+	 * @memberOf ClayTooltip
+	 * @param {!Event} event
+	 * @private
+	 */
+	_handleTouchStart(event) {
+		this._timer = setTimeout(() => {
+			const content = this._getContent(event.srcElement);
+			this._target = event.srcElement;
+
+			this._content = content;
+
+			if (!this.visible) {
+				this.element.style.display = 'block';
+			}
+			this._isTransitioning = true;
+			this.visible = true;
+			clearTimeout(this._timer);
+		}, 500);
+	}
+
+	/**
+	 * Handle touchend events.
+	 * @memberOf ClayTooltip
+	 * @param {!Event} event
+	 * @private
+	 */
+	_handleTouchEnd(event) {
+		if (this._timer) {
+			clearTimeout(this._timer);
+			this._timer = undefined;
+		}
+		if (this.visible) {
```

Comment:
This is the same code as present in _handleMouseLeave. You can extract it to a method _hideTooltip

Reply:
makes sense @carloslancha!